corehq/apps/hqwebapp/static/cloudcare/scss/formplayer-webapp/form.scss

```diff
@@ -263,3 +263,10 @@ $group-indent: 15px;
   position: fixed;
   bottom: 35px;
 }
+
+.mtb-05em-for-hs {
```

Comment:
B5 convention would have this prefixed `.my-` where `y` means top and bottom.
https://github.com/twbs/bootstrap/blob/v5.0.2/site/content/docs/5.0/utilities/spacing.md?plain=1#L33
The `05em-for-hs` itself feels verbose but it's descriptive; I don't think it _needs_ to change and I don't have any immediate suggestions.